markets/perps-market/contracts/storage/PerpsMarketFactory.sol

```diff
@@ -101,4 +107,21 @@ library PerpsMarketFactory {
     function withdrawMarketUsd(Data storage self, address to, uint256 amount) internal {
         self.synthetix.withdrawMarketUsd(self.perpsMarketId, to, amount);
     }
+
+    function transferLiquidatedSynth(
```

Comment:
Hey @sunnyvempati can you tell me what this function is used for please. afaik no synth gets ever sold, all collaterals are distributed to stakers.

Reply:
as this function states, this is for withdrawing "usd".  we withdraw usd for many reasons (i.e to pay keepers, to pay out profits etc).  

also synths do get sold when paying for fees as we discussed.

Reply:
Alright so i guess this is where  `Price.Tolerance.DEFAULT` needs to be  `INFINITE`? And aren't profits already in snxUSD, so why do we need to use this quoter?

Reply:
no worries, trying hard to avoid another sip 378. This needs to be done, otherwise liquidators will once again be not compensated for updating the necessary feeds involved with the position being liquidated.